include/swift/Frontend/ReferenceDependencyKeys.h

```diff
@@ -0,0 +1,41 @@
+//===--- ReferenceDependencyKeys.h - Keys for swiftdeps files ---*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_FRONTENDTOOL_REFERENCEDEPENDENCYKEYS_H
+#define SWIFT_FRONTENDTOOL_REFERENCEDEPENDENCYKEYS_H
+
+#include "swift/Basic/LLVM.h"
+#include "llvm/ADT/ArrayRef.h"
+#include <string>
+#include <vector>
```

Comment:
Not that it really matters, but the only includes you really need are LLVM.h and StringRef.h, no?

Reply:
Thanks!